bin/ember-template-lint.js

```diff
@@ -307,6 +308,14 @@ async function run() {
     return;
   }
 
+  if ((options.todoDaysToWarn || options.todoDaysToError) && !options.updateTodo) {
+    console.error(
+      'You must use `--update-todo` when using any of `--todo-days-to-warn` or `--todo-days-to-error`.'
+    );
```

Comment:
Lets tweak it slightly to:

> Using `--todo-days-to-warn` or `--todo-days-to-error` is only valid when the `--update-todo` option is being used.

Reply:
Y u no suggest?

Reply:
lol, :trollface: because it would fail your tests

Reply:
Done.